pyqtgraph/parametertree/parameterTypes/basetypes.py

```diff
@@ -256,6 +260,27 @@ def eventFilter(self, obj, ev):
         return self.callback(obj, ev)
 
 
+def el_value_to_dict(el: Element, attr = 'value') -> dict:
+    param_dict = {}
+    if attr in el.attrib:
+        value = el.get(attr)
+        if el.get('type') == 'bool':
```

Comment:
nit-pick but wonder if this would be a suitable case to use the match/case syntax that's new in Python 3.10.